python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -331,6 +333,9 @@ def __init__(self, config: BaseGaapFluxConfig, name, schema, logName=None) -> No
             flagName = self.ConfigClass._getGaapResultName(scalingFactor, "flag_gaussianization")
             flagDefs.add(flagName, "PSF Gaussianization failed when trying to scale by this factor.")
 
+        self.logName = logName
```

Comment:
You default this to `None` so are you happy for this to be `None`?

Reply:
The job of providing the `logName` is taken care of by the plugin framework. It actually can never handle `None` case within the plugin, which is why I had to give some string in the unit tests, where I run the plugin outside of the measurement framework. I thought about getting the default logger if `logName` is `None`. Should I go that way?

Reply:
Hmm. Okay, it's not clear from the parameter defaulting that this is impossible in general but it sounds like it's common knowledge for all plugin writers. I suppose a comment wouldn't hurt to remind people.

Reply:
This is now mentioned in the docstrings, where `logName` is defined.